plugin.video.svtplay/resources/__init__.py

```diff
@@ -0,0 +1 @@
+# Dummy to declare python package
```

Comment:
One note, when you stop supporting py2 and concentrate only on py3 you can safely remove all these empty init file placeholders. Python 3 doesn't require them